WordPressKit/WordPressKit/ReaderTopicServiceRemote+Subscription.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+
+
+extension ReaderTopicServiceRemote {
+    private struct Delivery {
+        static let frequency = "delivery_frequency"
+    }
+    
+    /// Subscribe action for site notifications
+    ///
+    /// - Parameters:
+    ///   - siteId: A site id
+    ///   - success: Success block
+    ///   - failure: Failure block
+    @nonobjc public func subscribeSiteNotifications(with siteId: NSNumber, _ success: @escaping () -> Void, _ failure: @escaping (NSError?) -> Void) {
```

Comment:
I'm not sure what's the point of all the `@nonobjc` here. And if this is swift only, would it make sense to use `Int` instead of `NSNumber`?